content/md/assets/assets-transformation.md

```diff
@@ -32,20 +32,42 @@ The following assets transformation are available:
     - `width`: maximum width in pixels
     - `height`: maximum height in pixels
 
+You can combine transformations, for instance, you can have a `scale` transformation and a `colorspace` transformation at the same time:
```

Comment:
Maybe say somewhere that it should be declared in yaml format, hence the format of the example. We only speak about the Yaml format in the sections after this paragraph.

Reply:
yes you are right, I'll add something above.